en/08_Changelogs/6.0.0.md

```diff
@@ -4,61 +4,63 @@ title: 6.0.0 (unreleased)
 
 # 6.0.0 (unreleased)
 
+Silverstripe CMS 6.0.0 is a major release. That means it contains many breaking changes and dependency upgrades. We tried our best to minimise upgrades woes, but you should allocate additional time to account for regression testing.
+
+This changelog provides a list of changes between Silverstripe CMS 5.4 and 6.0.
+
 ## Overview
 
 - [Change to commercially supported modules](#changes-to-support)
   - [Modules losing commercial support](#modules-losing-support)
+  - [`silverstripe/campaign-admin` no longer part of `silverstripe/recipe-cms`](#campaign-admin)
   - [New supported modules](#new-supported-modules)
+  - [New default front-end theme](#theme)
+  - [TinyMCE is in a separate module now](#tinymce)
+- [Dependency changes](#dependency-changes)
+  - [PHP version support](#php-version-support)
+  - [MySQL 5 no longer supported](#mysql-5-support)
+  - [`intervention/image` has been upgraded from v2 to v3](#intervention-image)
+  - [symfony dependencies have been upgraded from v6 to v7](#symfony)
+  - [JavaScript dependencies](#javascript-dependencies)
 - [Upgrade tools](#upgrade-tools)
 - [Features and enhancements](#features-and-enhancements)
   - [Validation added to DBFields](#dbfield-validation)
-  - [Changes to `sake`, `BuildTask`, CLI interaction in general](#cli-changes)
+  - [Changes to password validation](#password-validation)
+  - [Changes to `sake`, `BuildTask`, and CLI interaction in general](#cli-changes)
```

Comment:
1. Moved validation next to validation
2. Added "and" to "Changes to `sake`, `BuildTask`, and CLI interaction in general" to match actual heading